### ISSUE TITLE
DISCUSSION

### ISSUE THREAD

**Issue by epic-hacks** [epic-hacks/epic-hacks-surviv.io#1]
Since we are not officially released yet, lets have any discussions here.

**Comment by boo-tanki** [comment]
do u have 2 pay for extra, or is it all 4 free

**Comment by boo-tanki** [comment]
oh and, u should add, if u can, spawn w/ a gun or something, but the hack doesn't work rn

**Comment by boo-tanki** [comment]
![Screen Shot 2020-05-19 at 9 53 46 AM](https://user-images.githubusercontent.com/65607974/82335078-d1223900-99b6-11ea-8e49-6a63f83e396b.png)

**Comment by boo-tanki** [comment]
pls give me a better one!

**Comment by epic-hacks** [comment]
The hack is not yet released, that download is just for test, so please wait, we will release tomorrow or day after tomorrow.


> do u have 2 pay for extra, or is it all 4 free

We will be giving a free premium for everyone for a week after release. After that time the premium access is paid, and it is one-time payment for a lifetime access.
Keep in mind that it is a premium cheat, it is better than everything currently available for surviv. That's why we are giving a free week to try it all.

**Comment by riverzhao7** [comment]
Awesome hacks(By the look of it)

**Comment by riverzhao7** [comment]
How do you get premium?

**Comment by fire-and-ice582** [comment]
Is the hack going to be released today?

**Comment by epic-hacks** [comment]
> How do you get premium?

We will be going to give a premium for a week to everyone. Later you'll have to buy it only once.


> Is the hack going to be released today?

We were faced some issues yesterday related to the security of the EPIC-account(s). In order to provide a safe environment for everyone - we are changing our back-end, which shouldn't take more than a day. If everything go smooth - we are releasing today. Otherwise - tomorrow

**Comment by Forses** [comment]
Add a hack to the x16 sight and the house from spas-12 was visible on the map!

**Comment by riverzhao7** [comment]
you can add a gernade aimbot which basically gives aimbot to the gernade. Depending on distance between you and the enemy you can hit them with a gernade, but it has to explode just when the gernade touches the enemy. Warning: This is very hard

Enhancement

**Comment by epic-hacks** [comment]
> Add a hack to the x16 sight and the house from spas-12 was visible on the map!

I will work today on displaying house from spas-12 on the map. But about x16, it's impossible to adjust the scope to x16, coz if we do that, game doesn't send us all the objects beyond 1.3x ish of our current scope. But we can adjust the current scope by that 1.3 ish. I'll try to work that around

**Comment by epic-hacks** [comment]
> you can add a gernade aimbot which basically gives aimbot to the gernade. Depending on distance between you and the enemy you can hit them with a gernade, but it has to explode just when the gernade touches the enemy. Warning: This is very hard

I thought about that, but be aware that because of the grenades' slow flight time, aim would miss frequently, since player has enough time to change the vector of movement. So as for me - it would be impractical. What do you think?

**Comment by riverzhao7** [comment]
I think you could make it depending on where the person moves, if they go right you can turn it into a variable and then you can throw the gernade a little more right than the person so it will hit them dead on.

**Comment by epic-hacks** [comment]
@riverzhao7 that's true, but no one usually moves in a line.. that's what I'm concerned on. I will see what could be done there

**Comment by zhaoyangli** [comment]
for grenades, definitely there is a bug. When I throw it, it always go to the right side no matter where is the enemy.
so if aim bot is impossible for grenades just do not aim at all, let user control it.

**Comment by zhaoyangli** [comment]
another thing is melee, when you attack with melee, the bot should always follow the enemy like Icehack. Right now if I switch to melee I will be killed by a Icehack bot users.

**Comment by epic-hacks** [comment]
@zhaoyangli thank you for report. We will take care of grenade aimbot in the next update.

Regarding the melee, I was not implementing autofollow on purpose, since when using it, cheating is so obvious. 
The thing is that if you have a single weapon and ice-hacker approaches you, smartswitch takes care of a shoot and switch to melee, so if both of you have full health, you win coz you deal more damage to him.
I dont want to implement autofollow, BUT I'm already thinking of the way how to detect when enemy with autofollow approaches you (ice-hacker), and if so - cheat would take care of killing an ice-hacker. And I will definitely implement something like that in one of the upcoming updates

**Comment by zhaoyangli** [comment]
I spend lots of time playing with many bots. Your aim bot and smart switch is the best so far, great job!

For melee, now I never switch to melee then I can beat ice-hacker. For me it is fine not to fix it if it is too complicated.

Grenade thing is really affecting my game experience, please fix it first if it is possible.

and my premium access left is 727d3h19m, you said it is life time right?

**Comment by epic-hacks** [comment]
@zhaoyangli thank you for your support!

I have an idea how to deal with ice-hacks autoattack, i just need to do the math and debug it properly so that we don't have any false positives. So anti ice-hackers cheat hopefully would be ready soon.

Grenade would be fixed in the next update.

And premium. Yes, it's displaying as for 2 years, coz that way is easier for us to manage our database, but actually it is lifetime. I will update the display value probably in the next update as well, but no worries, we keep records of all purchases, so your purchase status would be updated as needed. If you purchased lifetime - you get lifetime. No scam from our side!

**Comment by zhaoyangli** [comment]
https://prnt.sc/st5mp3

https://prnt.sc/st5my8

funtime!

I am still not sure how to trigger the new melee attack feature.

**Comment by epic-hacks** [comment]
@zhaoyangli cool!
The new melee attack feature is not yet released, it still needs development :)

**Comment by Caioz10** [comment]
So, Can yall make the [twitch surviv](https://c79geyxwmp1zpas3qxbddzrtytffta.ext-twitch.tv/c79geyxwmp1zpas3qxbddzrtytffta/1.0.2/ce940530af57d2615ac39c266fe9679d/index_twitch.html?anchor=panel&language=en&mode=viewer&state=released&platform=web&popout=true) hacked?
